client/ayon_maya/plugins/create/create_look.py

```diff
@@ -37,6 +37,11 @@ def get_instance_attr_defs(self):
                     label="Convert textures to .rstex",
                     tooltip="Whether to generate Redshift .rstex files for "
                             "your textures",
+                    default=self.rs_tex),
+            BoolDef("includeTextureReferenceObjects",
+                    label="Texture Reference Objects",
+                    tooltip="Whether to use texture reference objects for "
+                            "your textures",
```

Comment:
Should this be more descriptive?
```suggestion
                    tooltip=(
                        "Whether to include texture reference objects "
                        "with the published look to reconnect to geometry "
                        "when assigning the look.",
```